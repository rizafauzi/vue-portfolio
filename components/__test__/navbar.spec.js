import { shallowMount } from '@vue/test-utils'
import Navbar from '../Navbar.vue'

describe('Navbar.vue', () => {
	let wrapper = null

	beforeEach(() => {
		wrapper = shallowMount(Navbar)
	})
	afterEach(() => {
		wrapper.destroy()
	})

	it('should render correctly', () => {
		
		expect(wrapper.element).toMatchSnapshot()
	})

	// it('should change text label when button clicked', () => {
	// wrapper.find('.btn-action').trigger('click.prevent');

	// expect(wrapper.vm.label).toBe('Hello');
	// })
})
