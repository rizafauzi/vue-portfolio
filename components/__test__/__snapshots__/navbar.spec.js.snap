// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar.vue should render correctly 1`] = `
<div
  class="container"
>
  <div
    class="navbar-container"
  >
    <div
      class="navbar-left"
    >
      <img
        alt="Playfest"
        src="~/static/playfest.png"
      />
    </div>
     
    <div
      class="navbar-right"
    >
      <img
        alt="Narasi"
        src="~/static/logo.png"
      />
    </div>
  </div>
</div>
`;
